canonical/lxd-ui
# Profile detail overhaul: header and overview tab [WD-2779]

## Done

- Profile detail page:
  - Header overhaul
  - Overview tab overhaul

Fixes [WD-2779](https://warthogs.atlassian.net/browse/WD-2779)

## QA

1. Run the LXD-UI:
    - On the demo server via the link posted by @webteam-app below. This is only available for PRs created by collaborators of the repo. Ask @lorumic or @edlerd for access.
    - With a local copy of this branch, run as described in the [Readme](https://github.com/canonical/lxd-ui#setting-up-for-development).
2. Perform the following QA steps:
    - Open a profile detail page
    - Check that the header is looking and behaving as expected
    - Check that the content of the overview tab is looking and behaving as expected - for regular projects as well as the default project

[WD-2779]: https://warthogs.atlassian.net/browse/WD-2779?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
